src/controllers/ImageTransformsController.php

```diff
@@ -121,15 +121,11 @@ public function actionSave(): ?Response
             $errors = true;
         }
 
-        if (!empty($transform->quality) && (!is_numeric($transform->quality) || $transform->quality > 100 || $transform->quality < 1)) {
```

Comment:
Due to the typing of `Transform::quality`, we can simply this condition as we know it is `?int`.